pkg/splunk/enterprise/util_test.go

```diff
@@ -1621,3 +1621,122 @@ func TestGetAppPackageLocalPath(t *testing.T) {
 		t.Errorf("Expected appPkgLocal Path %s, but got %s", expectedAppPkgLocalPath, calculatedAppPkgLocalPath)
 	}
 }
+
+func TestInitStorageTracker(t *testing.T) {
+	if operatorResourceTracker == nil {
+		t.Errorf("operatorResourceTracker should be initialized as part of the enterprise package init()")
+	}
+
+	// When the volume is not configured, should return an error
+	splcommon.AppDownloadVolume = "/non-existingDir"
+	err := initStorageTracker()
+	if err == nil {
+		t.Errorf("When the volume doesn't exist, should return an error")
+	}
+
+	// When the volume exists, should not return an error
+	splcommon.AppDownloadVolume = "/"
```

Comment:
for the testing purpose can we make this configurable value in either from configmap or from environment variable